ui/src/frontend/notes_panel.ts

```diff
@@ -311,6 +315,16 @@ export class NotesPanel extends Panel {
     globals.makeSelection(Actions.addNote({timestamp, color}));
   }
 
+  private onRightClick(x: number, _: number) {
+    if (x < 0 || !this.hoveredX) return;
+    for (const note of Object.values(globals.state.notes)) {
+      if (this.mouseOverNote(this.hoveredX, note) && note.noteType !== 'AREA') {
```

Comment:
Looks good to me. 

Just an additional thing that occurs to me. It's a nit, so act at your discretion. `mouseOverNote()` is not a trivial check; the `noteType` check, however, _is_ trivial. In many cases (but not all), you want to order a compound expression with the most trivial check first and the most expensive one last. I.e., you don't want to incur a heavy cost in one check when its outcome may be made moot by a subsequent, trivial one. Again, such a reordering may have no perceptible performance improvement here, so...your call.